_layouts/base.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="{{ page.lang | default: site.lang | default: 'en-US' }}">
+  <head>
+    {% include meta.html %}
+    {% include styles.html %}
+    {% include google_analytics.html %}
+    {% include dap.html %}
+    {% include google_recaptcha.html %}
+  </head>
+  {{ content }}
+  {% include scripts.html %}
```

Comment:
Problem: These scripts will be printed outside the body tag, which is not valid HTML:

![image](https://user-images.githubusercontent.com/1779930/100927263-a743e600-34b2-11eb-903e-271c94edfec0.png)

The `uswds-jekyll` theme addresses this by printing the `body` tag in the base layout, and allowing sub-layouts to add classes (etc) via YAML front-matter metadata.

Reply:
ah yikes, thanks for checking

Reply:
Fixed in 239663465e00077d45e9c2b8f1c0d873d7bca922